web_external/templates/body/studyPage.jade

```diff
@@ -10,8 +10,8 @@
   .isic-listing-content.isic-study-users-content
     if study.get('users').length
       ul
-        each user in study.get('users')
-          li= formatUser(user)
+        each user in study.users().toArray()
```

Comment:
`study.users().models` gives raw access to the collection's model array, which would be more efficient.